charts/karpenter-crd/Chart.yaml

```diff
@@ -2,8 +2,8 @@ apiVersion: v2
 name: karpenter-crd
 description: A Helm chart for AKS Karpenter provider Custom Resource Definitions (CRDs)
 type: application
-version: 0.1.0
-appVersion: 0.1.0
+version: 0-41f6dcabe655c7fec84f4467e95bbd75285ae082
```

Comment:
Assuming this will get reverted for PR completion

Reply:
Reverting. Neither value is great, sticking to the latest release probably makes the most sense for the in-repo chart. (Note that these values get automatically replaced for the published snapshot and release artifacts).